src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -52,6 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
 		String username = request.getParameter("username");
 		String password = request.getParameter("password");
+		String passwordHash = BCrypt.haspw(password,BCrypt.gensalt());
```

Comment:
In line 69 you forgot to change the 'password' parameter to 'passwordHash'

Reply:
Yup you're right, thanks

Reply:
BCrypt.haspw is not a method, should be hashpw

Also, you have not included the import statement for BCrypt in the file that is in the pull request.